code/modules/clothing/glasses/_glasses.dm

```diff
@@ -334,6 +334,17 @@
 	icon_state = "bigsunglasses"
 	inhand_icon_state = "bigsunglasses"
 
+/obj/item/clothing/glasses/sunglasses/mrbones
+	name = "nothing"
+	desc = "What are you wearing this for?"
+	icon_state = "mrbonesglasses"
+	inhand_icon_state = "bigsunglasses"
+	clothing_flags = BLOCKS_SHOVE_KNOCKDOWN

```

Comment:
I was debating telling you this, but I figured that I'm probably not gonna pick a (fair) fight with Mr. Bones anyway: BLOCKS_SHOVE_KNOCKDOWN does NOT protect you from the gun disarm that occurs when you get shoved twice in a row (even if you haven't been shoved into a wall), so a robust enough gamer/swarm of gamers could provoke Mr. Bones into pulling out his """.357""", then disarm him of it, then shoot him to death with it. Maybe the admins could roll with that, though, and make the new owner of the .357 the new Mr. Bones ("there must always be... a Mr. Bones") or something.